src/main/java/io/github/jbellis/brokk/gui/InstructionsPanel.java

```diff
@@ -286,6 +298,17 @@ public void changedUpdate(DocumentEvent e) {
         // Add this panel as a listener to context changes, only if CM is available
         this.contextManager.addContextListener(this);
 
+        // --- Autocomplete Setup ---
+        if (this.contextManager != null) {
```

Comment:
we don't need this. the comment saying that CM can be null is incorrect. should remove the @Nullable instead